Don't overwrite default opts in rich_text_area_tag (#43156)
rails/rails: actiontext/CHANGELOG.md

> *   Don't overwrite default opts in `rich_text_area_tag`
> 
>     You may want to use your own controller to authenticate requests or
>     perform server-side validations.

Reviewer: ```suggestion
*   Allow passing in a custom `direct_upload_url` or `blob_url_template` to `rich_text_area_tag`
```
